File: Client/ElectricityBillClient/src/app/Model/MeterAssign.ts
import { Customer } from './Customer';
import { MeterTable } from './MeterTable';

export class MeterAssign{
  MeterAssignId:number=0;
  CustomerId:number=0;
  MeterId:number=0;
  CreatedBy:string='';
  CreatedDate:Date|undefined;
  UpdatedBy:string='';
  UpdatedDate:Date|undefined;
  Status:number=0;
  // Customer: Customer = new Customer;
  // MeterTable: MeterTable = new MeterTable;
}

File: Server/ElectricityBillPayment/ModelClass/DTO/MeterAssign.cs
﻿using System;
using System.Collections.Generic;
using System.Text;
namespace ModelClass.DTO

{
    public class MeterAssign
    {
        public MeterAssign()
        {

        }
        public int MeterAssignId { get; set; }
        public int CustomerId { get; set; }
        public int MeterId { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public int Status { get; set; }
        public  Customer Customer { get; set; }
        public  MeterTable MeterTable { get; set; }
        public ICollection<MeterReadingTable> MeterReadingTable{get;set;}
    }

    public class MeterAssignSearch
    {
        public string Search { get; set; }
    }
}
